Source/MediaStorage.swift

```diff
@@ -236,7 +239,58 @@ public class MediaStorage {
             completionHandler(MediaContent(data: imageData), MediaStorageError(statusCode: nil, message: nil))
         }
     }
-
+    
+    public func addMeta(mediaId mediaId: String, userMeta: Dictionary<String, String>, completionHandler: MediaStorageError -> Void) {
+        if accessToken == nil {
+            completionHandler(
+                MediaStorageError(statusCode: nil, message: "wrong usage: use the connect method to get an access token.")
+            )
+            return
+        }
+        
+        if userMeta.isEmpty {
+            completionHandler(
+                MediaStorageError(statusCode: nil, message: "empty userMeta was taken: nothing to request.")
```

Comment:
To simplify the message, how about this one; "userMeta is empty: nothing to request."